### PR TITLE
Prometheus password fix

### PR DESCRIPTION
**Pull request by AKEB**
Prometheus password fix

## Description

Bug fix

## Motivation and Context

Fixed an issue where the password was not being used for prometheus metrics

## How has this been tested?

On my server

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by AKEB** [comment]
fromNodeMiddleware must be used before app.use(routerPrometheusMetrics); Otherwise it doesn't work

File: src/lib/Server.js [hunk 1]
```diff
@@ -335,22 +335,17 @@ module.exports = class Server {
...
-    app.use(routerPrometheusMetrics);
-
     // Check Prometheus credentials
     app.use(
       fromNodeMiddleware((req, res, next) => {
         if (!requiresPrometheusPassword || !req.url.startsWith('/metrics')) {
           return next();
         }
         const user = basicAuth(req);
-        if (requiresPrometheusPassword && !user) {
```
Comment (kaaax0815) [inline src/lib/Server.js:349]: Why did you remove this?

Comment (AKEB) [reply src/lib/Server.js:349]: This is unnecessary as there is already a check above

if (!requiresPrometheusPassword || !req.url.startsWith(‘/metrics’))

Comment (AKEB) [reply src/lib/Server.js:349]: OUCH. Although no!!!

**Comment by kaaax0815** [comment]
I also would appreciate a pr against `feat-ui`. As this will be the master if it's ready

File: src/lib/Server.js (hunk 1 above)
Comment (kaaax0815) [reply src/lib/Server.js:349]: Your right

**Comment by kaaax0815** [comment]
> I haven't dealt with this branch yet. I'm not sure I can do it quickly. And it's definitely not going to be today.

No worries

**Comment by kaaax0815** [comment]
@AKEB I think it makes most sense to do it on `feat-ui`. Which will probably/hopefully replace master by next release

**Comment by kaaax0815** [comment]
Is there any reason you wrapped numbers in `Number()`?
https://github.com/wg-easy/wg-easy/blob/3427d677f614ea5c81b49a8a449df605a0f56ad4/src/lib/WireGuard.js#L500-L502

**Comment by AKEB** [comment]
> Is there any reason you wrapped numbers in `Number()`?
> 
> https://github.com/wg-easy/wg-easy/blob/3427d677f614ea5c81b49a8a449df605a0f56ad4/src/lib/WireGuard.js#L500-L502

To make sure there's definitely a number

**Comment by AKEB** [comment]
> @AKEB I think it makes most sense to do it on `feat-ui`. Which will probably/hopefully replace master by next release

Sorry, but I don't have enough free time to study typescript and write code in two copies in two languages and under two frameworks.

**Comment by kaaax0815** [comment]
> Sorry, but I don't have enough free time to study typescript and write code in two copies in two languages and under two frameworks.

Unfortunate. Typescript is basically JavaScript and Nuxt also uses h3 under the hood (same as current api)

**Comment by kaaax0815** [comment]
lgtm. I'll merge it. Have to reimplement it either way
